cmd/skaffold/app/cmd/run.go

```diff
@@ -35,6 +35,7 @@ func NewCmdRun(out io.Writer) *cobra.Command {
 		},
 	}
 	AddRunDevFlags(cmd)
+	AddRunFlags(cmd)
```

Comment:
Do you think also adding the flag to `skaffold deploy` would make sense?

Reply:
Yah I think that makes sense, I can add it there as well